src/Microsoft.AspNet.Mvc/BuilderExtensions.cs

```diff
@@ -9,20 +9,29 @@
 
 namespace Microsoft.AspNet.Builder
 {
+    /// <summary>
+    /// Extension methods for <see cref="IApplicationBuilder"/> to add Mvc to the request execution pipeline.
+    /// </summary>
     public static class BuilderExtensions
     {
+        /// <summary>
+        /// Adds Mvc to the <see cref="IApplicationBuilder"/> request execution pipeline.
+        /// </summary>
+        /// <param name="app">The <see cref="IApplicationBuilder"/> to add Mvc too.</param>
+        /// <returns>The <paramref name="app"/>.</returns>
         public static IApplicationBuilder UseMvc([NotNull] this IApplicationBuilder app)
```

Comment:
Ehh, not super sure on this - remarks shouldn't be a place for specifying application design guidance. This sounds more like a sample \ doc thing